packages/edge-login-ui-rn/src/components/navigation/Router.tsx

```diff
@@ -23,9 +24,9 @@ import { LoadingScreen } from '../screens/LoadingScreen'
 import { NewAccountPasswordScreen } from '../screens/newAccount/NewAccountPasswordScreen'
 import { NewAccountPinScreen } from '../screens/newAccount/NewAccountPinScreen'
 import { NewAccountReviewScreen } from '../screens/newAccount/NewAccountReviewScreen'
+import { NewAccountTosScreen as NewAccountTOSScreen } from '../screens/newAccount/NewAccountTosScreen'
```

Comment:
Let's keep the name as it is, so `NewAccountTosScreen` everywhere instead of renaming it here in the import.